src/tiler/src/api.js

```diff
@@ -66,8 +66,9 @@ api.get(
         try {
             const { z, x, y } = processCoords(req)
             const layers = processLayers(req)
+            const { config } = req.queryString
```

Comment:
Destructuring assignment is cool, but for only one variable it seems like it the [Explicit is better than implicit]() principle kicks in and `const config = req.queryString.config` would be preferable.

Reply:
I agree, 100%. My reasoning for doing it like this was that a) eslint was yelling at me to do it like this and b) writing the assignment like this was a retroactive change to make the transition from this PR to the next commit (in the s3-config branch) more of an expansion on the same pattern, rather than a complete rewrite.